ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -28,6 +28,10 @@ tag_specification:
   name: "4.6"
   namespace: ocp
 tests:
+- as: unit
+  commands: cd go-controller && make test
+  container:
```

Comment:
Is this needed? can you take it out? maybe the src container has some go flags setup?

Reply:
I don't know. This is how unit tests are done elsewhere.